docs/content/preview/reference/configuration/yb-tserver.md

```diff
@@ -311,59 +311,52 @@ Default: `64`
 
 The number of shards per YB-TServer for each YCQL table when a user table is created.
 
-Default: `-1` (the value is calculated at runtime). For servers with up to two CPU cores, the default value is considered as `4`. For three or more CPU cores, the default value is considered as `8`. Local cluster installations, created with `yb-ctl` and `yb-docker-ctl`, use a value of `2` for this flag. Clusters created with `yugabyted` use a default value of `1`.
+Default: `-1` (the value based on the CPU cores is calculated at runtime).
```

Comment:
looks good, but I would go with shorter form `(the value is calculated at runtime)` as there's the detailed explanation one line below.

Reply:
correct , but it may be more validated if there's still a question /doubt for a user. So staying explicit in this case helps.